src/components/Footer.native.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import { connect, useDispatch } from 'react-redux'
+import * as pkg from '../../package.json'
+// import { TUTORIAL2_STEP_SUCCESS } from '../constants'
+import { logout, showModal } from '../action-creators'
+import { getSetting, isTutorial } from '../selectors'
+import { scaleFontDown, scaleFontUp } from '../action-creators/scaleSize'
+import { State } from '../util/initialState'
+import { View, StyleSheet, TouchableOpacity } from 'react-native'
+import { Text } from './Text.native'
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+const mapStateToProps = (state: State) => {
```

Comment:
I think we are moving away from this and should use useselector

Reply:
Converting to use useSelector 🏃🏾‍♂️